### ISSUE TITLE
Added Chinese language

### ISSUE THREAD

**Issue by cn-jiangfei** [iametza/ckeditor-html5-audio#18]
CKEDITOR.plugins.setLang(` 'html5video', 'cn', {
button: '发布HTML5视频',
title: 'HTML5视频',
infoLabel: '视频信息',
allowed: '支持格式: mp4, webm, ogv',
urlMissing: '视频url',
videoProperties: '视频属性',
upload: '上传视频',
btnUpload: '上传',
advanced: '高级',
autoplay: '自动播放',
yes: '是',
no: '否',
responsive: '自动宽度'
} );

**Comment by aldatsa** [comment]
Hi @cn-jiangfei,
this is the repository of the CKEditor HTML5 Audio plugin. You should create an issue in the repository of the CKEditor HTML5 Video plugin: https://github.com/bahriddin/ckeditor-html5-video/issues

If you want to contribute a translation for the Audio plugin as well, you should translate these strings. We will really appreciate your translation:

```
CKEDITOR.plugins.setLang( 'html5audio', 'en', {
    button: 'Insert HTML5 audio',
    title: 'HTML5 audio',
    infoLabel: 'Audio info',
    urlMissing: 'Audio source URL is missing.',
    audioProperties: 'Audio properties',
    upload: 'Upload',
    btnUpload: 'Send it to the server',
    advanced: 'Advanced',
    autoplay: 'Autoplay?',
    yes: 'Yes',
    no: 'No'
} );
```

**Comment by machinelll** [comment]
CKEDITOR.plugins.setLang( 'html5audio', 'cn', {
    button: '插入HTML5音频',
    title: 'HTML5 音频',
    infoLabel: '音频信息',
    urlMissing: '音频URL',
    audioProperties: '音频属性',
    upload: '上传',
    btnUpload: '上传到服务器',
    advanced: '高级',
    autoplay: '自动播放?',
    yes: '是',
    no: '否'
} );

**Comment by aldatsa** [comment]
Hi @machinelll,
thank you very much! We will include your translation in our next release.

**Comment by aldatsa** [comment]
Hi @machinelll,
we have just released a new version of our plugin (1.3.7) that includes your translation:
http://ckeditor.com/addon/html5audio
Best regards
